resources/keybinds.txt

```diff
@@ -36,8 +36,12 @@ KEYBINDINGS:
 [CHOP_WALLS:KEY_C]
 [CYCLE_TRACKING_MODE:KEY_F]
 [RESET_VIEW_OFFSET:KEY_Z]
-[DECR_SEGMENT_Z:KEY_1]
-[INCR_SEGMENT_Z:KEY_2]
+[DECR_SEGMENT_Z:KEYPAD_3]
+[INCR_SEGMENT_Z:KEYPAD_9]
```

Comment:
though I can see why you changed this, I'm not sure we should change the defaults. people can get *very* picky about their keybindings changing out from under them